object_mesh_path: '/catkin_ws/src/wros_tutorials/wrs/0000_examples/objects/milkcarton.stl'
gripper_name: 'robotiq140'
antipodal_grasp:
  angle_between_contact_normals: 90
  openning_direction: 'loc_x' 
  max_samples: 4
  min_dist_between_sampled_contact_points: .016
  contact_offset: .016 